src/BreadcrumbItem.js

```diff
@@ -55,14 +60,15 @@ const BreadcrumbItem = React.forwardRef(
   ) => {
     const prefix = useBootstrapPrefix(bsPrefix, 'breadcrumb-item');
 
-    const { href, title, target, ...elementProps } = props;
+    const { href, title, target, listItemProps, ...elementProps } = props;
     const linkProps = { href, title, target };
 
     return (
       <Component
         ref={ref}
         className={classNames(prefix, className, { active })}
         aria-current={active ? 'page' : undefined}
+        {...listItemProps}
```

Comment:
```suggestion
    const { href, title, target, ...props } = props;
    const linkProps = { href, title, target };

    return (
      <Component
        ref={ref}
        {...props}
        className={classNames(prefix, className, { active })}
        aria-current={active ? 'page' : undefined}
```

maybe we can just do this?

Reply:
would break backwards compatibility right? thats why I made this approach.

Reply:
we're in betas :D

post-https://github.com/react-bootstrap/react-bootstrap/pull/5043 i think it's a reasonable change to make...

Reply:
maybe we should release v1 soon, so that we have at least something for Bootstrap 4.

Then for Bootstrap 5 we can use v2 or instantly jump to v5 idk. (seems to be the user friendly approach for me)

Reply:
yeah... i guess my concern with matching bootstrap major versions is that it doesn't let us make potentially necessary breaking changes on our end.

part of the question might be how often upstream cuts breaking changes. if it will happen on a semi-regular cadence, then not too big a deal.

if it will be years and years, then we have a bit of a problem